actions/create_instance.yaml

```diff
@@ -30,6 +30,16 @@ parameters:
     type: "string"
     description: "SSH key to use during intial instance creation"
     required: false
+  BlockDeviceMappings:
+    type: "array"
+    items:
+      type: "object"
+    description: "The block device mapping entries"
+    required: false
+  IamInstanceProfile:
+    type: "object"
```

Comment:
Same here - probably also need to default it to an empty object so it works if this parameter is not provided.

Would be good to test those scenarios as well (making sure it works correctly when those two parameters are not provided).